pkg/lib/packagekit.es6

```diff
@@ -244,7 +244,8 @@ export function cancellableTransaction(method, arglist, progress_cb, signalHandl
                         // avoid calling progress_cb after ending the transaction, to avoid flickering cancel buttons
                         ErrorCode: (code, detail) => {
                             progress_cb = null;
-                            reject({ detail, code: cancelled ? "cancelled" : code });
+                            reject({ detail, code: cancelled ? "cancelled" : code,
+                                     toString: () => detail });
```

Comment:
That's a nice trick, thanks!

Reply:
It's obsolete now with the use of TransactionError, right?